Gemfile.lock

```diff
@@ -237,72 +238,72 @@ GEM
       timeout
     net-protocol (0.1.3)
       timeout
-    net-scp (3.0.0)
-      net-ssh (>= 2.6.5, < 7.0.0)
+    net-scp (4.0.0.rc1)
+      net-ssh (>= 2.6.5, < 8.0.0)
     net-smtp (0.3.1)
       digest
       net-protocol
       timeout
-    net-ssh (6.1.0)
+    net-ssh (7.0.1)
```

Comment:
### [net-ssh](https://github.com/net-ssh/net-ssh)

**Major** version upgrade :chart_with_upwards_trend::exclamation: 6.1.0 → 7.0.1

[_[change-log](https://github.com/net-ssh/net-ssh/blob/master/CHANGES.txt), ~~source-code~~, [gem-diff](https://my.diffend.io/gems/net-ssh/6.1.0/7.0.1)_]





<details>
<summary>Commits</summary>

A change of **175** commits. See the full changes on [the compare page](https://github.com/net-ssh/net-ssh/compare/v6.1.0...v7.0.1).

These are the first 10 commits:
- (2938f38) [Add ssh agent lock and unlock](https://github.com/net-ssh/net-ssh/commit/2938f3883f294334915e196ca635b223c8f964a7)
- (bff0dcf) [Correct typo in comment](https://github.com/net-ssh/net-ssh/commit/bff0dcf66dd2e3e888fac00c9bb2f166af341346)
- (be1708c) [Tweak max constant value to work with JRuby 9.1](https://github.com/net-ssh/net-ssh/commit/be1708c5fef78886319e8d10d201aece53bc1b0f)
- (e9239b7) [Re-add logic for JRuby](https://github.com/net-ssh/net-ssh/commit/e9239b7e035ec6adde65f0e48672c5f7a99cad1f)
- (ff8c18d) [Switch back to ecb](https://github.com/net-ssh/net-ssh/commit/ff8c18d13cae12092131b20503783c8274583f50)
- (f2223f0) [Try to fix CI](https://github.com/net-ssh/net-ssh/commit/f2223f03ff3552a2b6de8364c12b0d41258742c1)
- (a65c734) [Update rubocop-todo.yml](https://github.com/net-ssh/net-ssh/commit/a65c7348ad6b3dca4da299d7726588b543016707)
- (d0b4ffe) [Fix Layout/TrailingBlankLines](https://github.com/net-ssh/net-ssh/commit/d0b4ffe8984dafbfafd79ac99caef4229edb8557)
- (ee26aad) [Fix Style/DefWithParentheses](https://github.com/net-ssh/net-ssh/commit/ee26aad6bb4f7c1c53a140889478b73d34daf4d5)
- (c8d50d7) [Merge pull request #769 from net-ssh/fix-cdn-for-ci](https://github.com/net-ssh/net-ssh/commit/c8d50d77d88b2b0986d1d31404e67add76136b55)


</details>